src/components/new-request/submit-request/entity-cannot-be-auto-analyzed.vue

```diff
@@ -74,11 +74,7 @@ export default class EntityCannotBeAutoAnalyzed extends Vue {
     }
   }
 
-  get nameAnalysisTimedOut () {
-    return this.getNameAnalysisTimeout
-  }
-
-  get boxes () {
+  get boxes (): any {
```

Comment:
I added this generic type to pre-empt template warnings. (In the template, `boxes` had type "never", causing de-reference warnings.)